Screens/ParkedCarForm.jsx

```diff
@@ -7,47 +7,104 @@ import {
   View,
 } from "react-native";
 import { Picker } from "@react-native-picker/picker";
-import { useContext, useState } from "react";
+import { useContext, useState, useCallback } from "react";
 import UploadImage from "../Components/UploadImage";
 import { getCurrentDateTime } from "../utils/utils";
 import { ParkingFormModal } from "../Components/Modals/ParkingFormModal";
+import { FIRESTORE_DB } from "../firebaseConfig";
+import { Timestamp, collection, doc, addDoc, setDoc, query, orderBy, limit, getDocs } from "firebase/firestore";
+
 
 const ParkedCarForm = () => {
   const [duration, setDuration] = useState("60");
   const [reminder, setReminder] = useState("5");
   const [notes, setNotes] = useState("");
 
+  const handleSubmit = useCallback(async () => {
+    try {
+      const parkedCarsCollection = collection(FIRESTORE_DB, "parkings");
+
+      const parkDoc = {
+        duration,
+        reminder,
+        notes,
+        timestamp: Timestamp.now(),
+      };
+
+      const { id } = await addDoc(parkedCarsCollection, parkDoc);
+
+
+
+      console.log("Data submitted successfully!");
+    } catch (error) {
+      console.error("Error submitting data:", error);
+    }
+  }, [duration, reminder, notes]);
+
   return (
-    <KeyboardAvoidingView>
-      <Text>ParkedCarForm</Text>
+    <KeyboardAvoidingView style={styles.container}>
+      <Text style={styles.title}>ParkedCarForm</Text>
       <UploadImage />
-      <Text>Form incoming {getCurrentDateTime()}</Text>
-      <Text>I'm going to park</Text>
+      <Text style={styles.subtitle}>Form incoming {getCurrentDateTime()}</Text>
+      <Text style={styles.label}>I'm going to park</Text>
       <Picker
         selectedValue={duration}
         onValueChange={(itemValue, itemIndex) => setDuration(itemValue)}
+        style={styles.picker}
       >
         <Picker.Item label="30 minutes" value="30" />
         <Picker.Item label="1 hour" value="60" />
         <Picker.Item label="2 hours" value="120" />
         <Picker.Item label="3 hours" value="180" />
       </Picker>
-      <Text>Remind me</Text>
+      <Text style={styles.label}>Remind me</Text>
       <Picker
         selectedValue={reminder}
         onValueChange={(itemValue, itemIndex) => setReminder(itemValue)}
+        style={styles.picker}
       >
         <Picker.Item label="5 minutes before" value="5" />
         <Picker.Item label="10 minutes before" value="10" />
         <Picker.Item label="15 minutes before" value="15" />
         <Picker.Item label="30 minutes before" value="30" />
       </Picker>
       <ParkingFormModal notes={notes} setNotes={setNotes} />
```

Comment:
I can't see the notes when the popup is closed? I wonder if using some kind of text field would be better?

Reply:
This can easily be populated from the modal so the user can see there notes and then submit. As above, there has been minimal styling and this ticket only covers pushing data to the db.